src/legacy/keyboardEvent.js

```diff
@@ -17,9 +17,7 @@
         const KeyboardEvent = function (eventName, options) {
             options = options || {};
 
-            const event = document.createEvent('Event');
-
-            event.initEvent(eventName, !!options.bubbles, !!options.cancelable);
+            const event = new Event('Event', { bubbles: !!options.bubbles, cancelable: !!options.cancelable });
```

Comment:
```suggestion
            const event = new Event(eventName, { bubbles: !!options.bubbles, cancelable: !!options.cancelable });
```

Reply:
Oops, sorry about that! I have updated this commit with the two changes you suggested.